direct color picker

peroperje opened the pull request:
Need to update dependencies

- Added https://casesandberg.github.io/react-color/. and <Github /> as the color picker
- At the form,  I've named field with color picker  "category" as it defined at the picture
- Added colored square on Direct list  
One suggestion:
Maybe it is better to put color as the background color of avatar instead current solution.

VGraupera commented:
A couple of comments.

1/ OK, I agree. Make the background of the avatar match the color instead of the current solution. If the color, however, is either very dark or very light can you still make the letters in the avatar visible?  

2/ Set color. Save record. Go back to record and clear the color. The update button is still disabled. It should be enabled to save the change of color.

Thanks.

peroperje commented:
1. Added the category color as the background color of the avatar.
I've added  TinyColor library (https://www.npmjs.com/package/tinycolor2) that decide font colors(light or dark) by background color 
Need to update dependencies
2. the bug has been fixed
